tools/ovc/openvino/tools/ovc/cli_parser.py

```diff
@@ -25,9 +26,18 @@
 # Helper class for storing input cut information
 _InputCutInfo = namedtuple("InputCutInfo", ["name", "shape", "type", "value"], defaults=[None, None, None, None])
 
+
 def is_shape_type(value):
```

Comment:
let us have these utils functions in `utils` file because `cli_parser` is quite overloaded. Please check if we have already such routine because we have already implemented quite similar logic that relates also to types.